### PR TITLE
Feature/fix fetch

### PR DESCRIPTION
**Pull request by nware1066**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features
### Detailed Description
Working on bug, we are attempting to solve problems with retrieving and displaying the data from API
### Files modified:
- [ ] index.html
- [ ] style.scss
- [ ] scripts.js
- [ ] domUpdates.js
- [x] Other files:

### Message/Questions for reviewer:

### CODE REVIEW

File: src/Components/CardSection.js [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
-      .catch(error => console.log('Movie not found'))
-  }
 
-  render() {
-    console.log('lives in card section', this.state.movies)
-    const allMovies = this.state.movies.map(movie => {
+function CardSection(props) {
+  let movies;
+  console.log('length', props.allMovies.movies)
+  if (props.allMovies.length > 0) {
```
Comment (Leasw144) [inline src/Components/CardSection.js:8]: I recall Leta doing something like a shorthand if conditional using &&, does anyone think that may be relevant here?

Comment (H-Bo214) [inline src/Components/CardSection.js:8]: I'll get with my mentor about the issues concerning this if conditional.

File: src/Components/CardSection.js [hunk 2]
```diff
@@ -1,35 +1,31 @@
...
+    console.log('test', props)
+    movies = props.allMovies.movies.map(movie => {
       return (
-      <MovieCard movie={movie} />
-    )
-  })
-    return (
-      <section>
```
Comment (Leasw144) [inline src/Components/CardSection.js:27]: I think that we can refactor our map to be directly placed within the return statement when we render the card section. Something to think of once we figure out our current issue.

**Comment by Leasw144** [comment]
Thanks for driving today!

**Pull request merged**